templates/emails/block-email-template.php

```diff
@@ -23,7 +24,7 @@
 
 		body {
 			background-color: #f3f3f3;
-			font-family: -apple-system, system-ui, blinkmacsystemfont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+			font-family: <?php echo esc_attr( $sensei_font_family ); ?>;
```

Comment:
I think we can use CSS variables instead of PHP variables.

Reply:
I was thinking the same too, but emogrifier has doesn't support variables out of the box it seems :( Have to try to take care of it under the global style card